packages/grafana-ui/tsconfig.json

```diff
@@ -1,7 +1,6 @@
 {
   "compilerOptions": {
     "baseUrl": "node_modules/@types",
-    "declarationDir": "dist",
```

Comment:
This will make the dist package not include declaration files, so it can not be removed unless we programmatically move all *.d.ts files when the package is bundled.